source/renderer/app/components/splash/SplashNetworkITN.js

```diff
@@ -76,6 +79,7 @@ export default class SplashNetworkITN extends Component<Props> {
         description={description}
         buttonLabel={buttonLabel}
         linkLabel={linkLabel}
+        currentTheme={currentTheme}
```

Comment:
@DeeJayElly why do we need to pass the currentTheme here?

Reply:
We need to recheck the current theme as the background for non incentivized themes should be plain without circles.